code/game/gamemodes/miniantags/mosquito/mosquito.dm

```diff
@@ -0,0 +1,89 @@
+/mob/living/simple_animal/mosquito
+	name = "Mosquito"
+	desc = "A blood thirsty beast, the carrier of diseases"
+	icon_state = "mosquito_alive"
+	icon_living = "mosquito_alive"
+	icon_dead = "mosquito_dead"
+	icon = 'icons/mob/mosquito.dmi'
+	turns_per_move = 0
+	attacktext = "stings"
+	response_help  = "shoos"
+	response_disarm = "swats away"
+	response_harm   = "squashes"
+	maxHealth = 10
+	health = 10
+	faction = list("hostile")
+	speed = 0
+	obj_damage = 0
+	environment_smash = 0
+	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
+	ventcrawler = VENTCRAWLER_ALWAYS
+	mob_size = MOB_SIZE_TINY
+	mob_biotypes = MOB_ORGANIC | MOB_BUG
+	blood_volume = BLOOD_VOLUME_NORMAL
+	flying = TRUE
+	deathmessage = "Stops flapping it's wings!"
+	var/list/donors = list()
+	var/playstyle_string = 	"<b><font size=3 color='red'>You are a giant mosquito.</font><br> The most annoying being in the universe and \
+							a carrier for diseases.<br> Your goal is to take blood from biological beings, feeding and healing yourself \
+							in the process <br> "
+	var/datum/disease/DiseaseOfM
+
+/mob/living/simple_animal/mosquito/Initialize(mapload,ChosenDisease)
+	. = ..()
+	remove_from_all_data_huds()
+	GrantMosquitoActions()
+	DiseaseOfM = new ChosenDisease
```

Comment:
game_logic_variables_and_procs_should_use_[snake_case](https://en.wikipedia.org/wiki/Snake_case)

Reply:
also won't this produce a runtime if you spawn one of these from the game panel?

Reply:
It will, any suggestions on how to fix this?

Reply:
I'd remove that from the argument and just set it manually when creating the mosquitoes in the event.

Alternatively, make the mosquito randomly decide what the disease is when they're spawned.

Reply:
could make it an optional param as well. If it's given then use it. If not generate a random one